themes/classic/templates/checkout/order-confirmation.tpl

```diff
@@ -6,11 +6,11 @@
 
 {block name='page_content_container' prepend}
   <section id="content-hook_payment_return">
-    {$HOOK_PAYMENT_RETURN}
+    {$HOOK_PAYMENT_RETURN|escape:'html':'UTF-8'}
   </section>
 
   <section id="content-hook_order_confirmation">
-    {$HOOK_ORDER_CONFIRMATION}
+    {$HOOK_ORDER_CONFIRMATION|escape:'html':'UTF-8'}
```

Comment:
I'm not realy sure, because I can't assume which module(s) will use and override the rendering of this block :/